draft-ietf-privacypass-architecture.md

```diff
@@ -192,8 +216,10 @@ Token Response, the Client computes a token from the token challenge and Token
 Response. This token can be validated by anyone with the per-Issuer key, but
 cannot be linked to the content of the Token Request or Token Response.
 
-6. If the Client has a token, it includes it in a subsequent request to
-   the Origin, as authorization. This token is sent only once.  The Origin
+6. If the Client has a token, it includes it in a subsequent, retried request to
```

Comment:
I don't think the request is necessarily a "retried" one — it's a new request with a new header in it. It's in reaction to the challenge, certainly, but I don't know if "retried" is quite right.

Reply:
Hm, true, I suppose it doesn't need to be the exact same request. Will revert.